roles/dtc/common/tasks/sub_main.yml

```diff
@@ -157,4 +143,32 @@
 # --------------------------------------------------------------------
 
 - name: Build Fabric interface All List From Template
-  ansible.builtin.include_tasks: ndfc_interface_all.yml
\ No newline at end of file
+  ansible.builtin.include_tasks: ndfc_interface_all.yml
+
+- name: Run Diff Flags
+  ansible.builtin.debug:
+    msg:
+      - "----------------------------------------------------------------"
+      - "+     Fabric Changes Detected -               [ {{ changes_detected_fabric }} ]"
+      - "+     Inventory Changes Detected -            [ {{ changes_detected_inventory }} ]"
+      - "+     vPC Link Peer Changes Detected -        [ {{ changes_detected_link_vpc_peering }} ]"
+      - "+     vPC Peer Changes Detected -             [ {{ changes_detected_vpc_peering }} ]"
+      - "+     ----- Interfaces -----"
+      - "+     Interface vPC Changes Detected -        [ {{ changes_detected_interface_vpc }} ]"
+      - "+     Interface Access Changes Detected -     [ {{ changes_detected_interface_access }} ]"
+      - "+     Interface Access PO Changes Detected -  [ {{ changes_detected_interface_access_po }} ]"
+      - "+     Interface Loopback Changes Detected -   [ {{ changes_detected_interface_loopback }} ]"
+      - "+     Interface PO Routed Changes Detected -  [ {{ changes_detected_interface_po_routed }} ]"
+      - "+     Interface Routed Changes Detected -     [ {{ changes_detected_interface_routed }} ]"
+      - "+     Interface Trunk Changes Detected -      [ {{ changes_detected_interface_trunk }} ]"
+      - "+     Interface Trunk PO Changes Detected -   [ {{ changes_detected_interface_trunk_po }} ]"
+      - "+     Sub Interface Routed Changes Detected - [ {{ changes_detected_sub_interface_routed }} ]"
+      - "+     ----- All Interfaces -----"
+      - "+     All Interfaces Changes Detected -       [ {{ changes_detected_interfaces }} ]"
+      - "+     ----- All Interfaces -----"
+      - "+     VRFs Changes Detected -                 [ {{ changes_detected_vrfs }} ]"
+      - "+     Networks Changes Detected -             [ {{ changes_detected_networks }} ]"
+      - "+     ----- Run Map -----"
+      - "+     Run Map Diff Run -                      [ {{ run_map_read_result.diff_run }} ]"
+      - " +    Force Run Flag  -                       [ {{ force_run_all }} ]"
```

Comment:
@mikewiebe When force_run_all is not defined in group_vars, it is erroring out for the playbook.  I don't see the default value defined in this PR.

Reply:
@juburnet Good catch.  It was a new file and I forgot to add it to the diffs.  Added now.

https://github.com/netascode/ansible-dc-vxlan/pull/172/files#diff-7584e1953f921f9c0b101993010aaae7ac16d33d25c3d5a2ceed4d693b5af351